.ocamlformat-ignore

```diff
@@ -1,2 +1,3 @@
 boot/libs.ml
 src/dune_rules/assets.ml
+vendor/**
```

Comment:
Can you make this a separate change? We already have an ocamlformat ignore in vendor, but it doesn't cover all the projects.

Reply:
Ah I missed that file. Here's a PR that adds opam https://github.com/ocaml/dune/pull/7744